### PR TITLE
[1LP][RFR] move the metaplugins folder into cfme

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
this moves the metaplugins package into cfme

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit aa195015f4d1bbb87bcd531f208a0972a239dae4:

`fixtures/blockers.py`:
- :bangbang: Line 5:101: **E501** *line too long (103 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: utils/apidoc.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 # When adding/removing from this list, remember to edit docs/modules.rst to match
```
Comment (psav) [inline utils/apidoc.py]: Comment should be adhered to I think

File: fixtures/blockers.py [hunk 2]
```diff
@@ -1,12 +1,14 @@
 # -*- coding: utf-8 -*-
 """Collection of fixtures for simplified work with blockers.
 
-You can use the :py:func:`blocker` fixture to retrieve any blocker using blocker syntax (as
-described in :py:mod:`metaplugins.blockers`). The :py:func:`bug` fixture is specific for bugzilla,
-it accepts number argument and spits out the BUGZILLA BUG! (:py:class:`utils.bz.BugWrapper`, not the
-:py:class:`utils.blockers.BZ` instance!). The :py:func:`blockers` retrieves list of all blockers
-as specified in the meta marker. All of them are converted to the :py:class:`utils.blockers.Blocker`
-instances
```
Comment (psav) [inline fixtures/blockers.py]: Why reformatting all this?

Comment (RonnyPfannschmidt) [inline fixtures/blockers.py]: some refs  changes and broke the formatting vs line-length, so i fixed the flow

File: utils/apidoc.py (hunk 1 above)
Comment (RonnyPfannschmidt) [inline utils/apidoc.py]: good catch

**Pull request merged**